packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/dedicatedCloud-dashboard.routes.js

```diff
@@ -0,0 +1,7 @@
+import component from './dedicatedCloud-dashboard.component';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.dedicatedClouds.dashboard', {
+    component,
```

Comment:
No need to use the angular declared component ?

Reply:
Good point, I'll check

Reply:
It was unused so I removed the declaration ! Good find @marie-j